docs/mast/mast.rst

```diff
@@ -877,6 +877,14 @@ Given an HSC Match ID, return all catalog results.
    410574498 63980492 ...   -1.10056e-005 1.56577e-009 1.56577e-009 1.10056e-005
    410574497 63980492 ...   -1.10056e-005 1.56577e-009 1.56577e-009 1.10056e-005
 
+"MatchID" is case sensitive in this case, and will produce an error with different
+capitalization.
+
+.. doctest-skip::
```

Comment:
Do not skip these tests, doctest can handle checking exceptions and warnings, you need to insert the actual error you see to make the test pass (there are some examples scattered over the docs)